core/domain/exp_domain.py

```diff
@@ -2762,6 +3617,16 @@ def _migrate_to_latest_yaml_version(
     def from_yaml(cls, exploration_id, yaml_content):
         """Creates and returns exploration from a YAML text string for YAML
         schema versions 10 and later.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            yaml_content: str. The YAML representation of the exploration.
+
+        Returns:
+            Exploration. The corresponding exploration domain object.
+
+        Raises:
+            Exception: Incorrect YAML version.
```

Comment:
Could you give a bit more detail about what's happening here? What has the incorrect YAML version? (Please read the method, and the methods it calls, to understand what's actually going on.)